### PR TITLE
JOT-190 add role as parameter

### PR DESCRIPTION
**Pull request by cmurat**
I am creating automated tests for Jira DC template on the Jira side and the AWS account I am using shouldn't be able to create a role. Thus, I need to pass an existing role.

### CODE REVIEW

**Review by tarka — APPROVED** [review]

**Comment by theghostwhoforks** [comment]
Hey @cmurat 

Can you raise a PR to point to develop instead of master? Our master branch follows upstream aws-quickstarts (https://github.com/aws-quickstart/quickstart-atlassian-jira/tree/master). Once a PR is merged, we will raise a PR to upstream develop. AWS will, at a later point of time, merge to master and then release the template.

**Comment by cmurat** [comment]
@theghostwhoforks Thanks! I changed the target of the PR.

**Comment by cmurat** [comment]
I have removed the default value I added.

**Review by theghostwhoforks — APPROVED** [review]

**Comment by theghostwhoforks** [comment]
@nanux Can you review and merge this PR (when you're back)?

**Review by bpar476 — APPROVED** [review]
Just deployed and tested this version of the template and it looks like it's all working well :)

**Comment by nanux** [comment]
@cmurat, do you still plan to go forward with this change?

**Comment by cmurat** [comment]
Hi @nanux, sorry for the late response. I didn't see the mail as this is my personal account. 

Yes, we still need this. Jira CI is using my branch currently.

**Comment by cmurat** [comment]
@nanux I just resolved the conflicts. Can you take a look and merge if it looks good?

**Comment by bpar476** [comment]
@cmurat Adam is on leave at the moment, I'll take a look.

File: templates/quickstart-jira-dc.template.yaml
Comment (bpar476) [inline templates/quickstart-jira-dc.template.yaml:499]: I don't know if this description is helpful to users in the UI. They don't really see the resources. `JiraClusterNodeRoleDefault` will probably mean nothing to them. I think we need to make it clear it's optional and for advanced configuration only.

@ddomingorh what do you think

File: templates/quickstart-jira-dc.template.yaml
Comment (bpar476) [inline templates/quickstart-jira-dc.template.yaml]: I feel like this should be at the bottom of the `Cluster nodes` group because it is not a very important parameter.

**Review by bpar476 — CHANGES_REQUESTED** [review]

File: templates/quickstart-jira-dc.template.yaml
Comment (cmurat) [reply templates/quickstart-jira-dc.template.yaml]: Done

File: templates/quickstart-jira-dc.template.yaml
Comment (cmurat) [reply templates/quickstart-jira-dc.template.yaml:499]: Can you take a look at the new description?

**Comment by cmurat** [comment]
@bpar476 can you look at the PR again?

File: templates/quickstart-jira-dc-with-vpc.template.yaml
Comment (bpar476) [inline templates/quickstart-jira-dc-with-vpc.template.yaml:682]: What's the purpose of this addition?

Comment (cmurat) [reply templates/quickstart-jira-dc-with-vpc.template.yaml:682]: `quickstart-vpc-for-atlassian-services.yaml ` uses these parameters. When they are not passed, the template looks for the default bucket and fails.

**Comment by bpar476** [comment]
@cmurat Are you able to synchronise your branch with develop? I want to run our builds against it and it's hitting an old linter error

**Comment by cmurat** [comment]
> @cmurat Are you able to synchronise your branch with develop? I want to run our builds against it and it's hitting an old linter error

@bpar476 It is synchronised now.

**Pull request closed without merging**
